src/Symfony/Component/Messenger/Stamp/FutureStamp.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+use Amp\Future;
+
+/**
+ * This stamps allows passing the future representing the potential result of the handler,
```

Comment:
```suggestion
 * This stamp allows passing the future representing (....?) the potential result of the handler,
```

Reads weird the sentence, is there something missing in `(....?)`?